src/main/components/RequestBar.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useContext } from 'react';
```

Comment:
I'm confused. Is this revised component file the one being rendered?
If we're rendering the one in xcomponents, there shouldn't be any changes in this PR to this component.

Reply:
The original goal was to modify RequestBar.jsx, but it was all context API and I decided  to create a new file under xcomponents directory.  Next time when I add files I ll use git add -p instead of git add .